mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -166,16 +166,19 @@ class SetpointRawPlugin : public plugin::PluginBase,
 			position = ftf::transform_frame_baselink_aircraft(position);
 			velocity = ftf::transform_frame_baselink_aircraft(velocity);
 			af = ftf::transform_frame_baselink_aircraft(af);
+			yaw = ftf::quaternion_get_yaw(
+					ftf::quaternion_from_rpy(M_PI, 0.0, 0.0)*(
```

Comment:
Please use the `ftf::transform_orientation_aircraft_baselink` instead.

Reply:
With transform_orientation_aircraft_baselink, that is, the original multiplication order, the roll is flipped after the rotation, making no effect on yaw.

Reply:
Oh I see. So can you instead create a new method in frame_tf to handle this transform?

Reply:
on it.

Reply:
You are still not applying the new funtions